Service/Runner.php

```diff
@@ -297,21 +292,31 @@ public function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId)
      * @param $action
      * @param $mode
      * @param $jobId
-     * @return Output[]
+     * @param array $jobParams
+     * @return array
      */
-    public function run(array $jobDefinitions, $action, $mode, $jobId)
+    public function run(array $jobDefinitions, $action, $mode, $jobId, array $jobParams = [])
     {
         if (count($jobDefinitions) > 1 && $mode != 'run') {
             throw new UserException('Only 1 row allowed for sandbox calls.');
         }
         $outputs = [];
         foreach ($jobDefinitions as $jobDefinition) {
             if ($jobDefinition->isDisabled()) {
-                $this->loggerService->getLog()->notice(
-                    "Skipping configuration id: " . $jobDefinition->getConfigId() . ' version:' . $jobDefinition->getConfigVersion()
-                    . ", rowId: " . $jobDefinition->getRowId()
-                );
-                continue;
+                if (count($jobDefinitions) === 1 && isset($jobParams['row']) && $jobParams['row'] === $jobDefinition->getRowId()) {
```

Comment:
potrebujem na neco podminku count($jobDefinitions) === 1  ?

Reply:
Doublecheck, aby bylo jasný, že to pustí pouze jeden řádek. Ještě by se týhle metodě mohl místo `JobDefinition[]` předat `JobDefinitionParser` a tím by tahle podmínka odpadla, protože by si tu `Runner` sám odfiltroval. 

Akorát to bude děsná mrdačka to přepsat v testech. Kdyby se to spláchlo s vymazáním `$mode` a předáváním celejch `$jobParams`, tak by to asi bylo snesitelnější.

Reply:
predat jobParams me taky npaadlo, ale zase jsou pak vsude rozlezly klice jobparams - ze to proste probublava kamsi do strev

Reply:
A přesunul jsem filtrování `JobDefinitions[]` do `Runner`u, nějak to asi dává větší smysl.